packages/nimble-components/src/rich-text/editor/template.ts

```diff
@@ -84,12 +93,33 @@ export const template = html<RichTextEditor>`
                         ${x => richTextToggleNumberedListLabel.getValueFor(x)}
                         <${iconNumberListTag} slot="start"></${iconNumberListTag}>
                     </${toggleButtonTag}>
+                    ${repeat(x => x.getMentionExtensionConfig(), html<MentionExtensionConfiguration, RichTextEditor>`
+                        <${buttonTag}
+                            appearance="ghost"
+                            content-hidden
+                            ?disabled="${(_x, c) => c.parent.disabled}"
+                            slot="start"
+                            title=${x => x.buttonLabel}
```

Comment:
This is because we haven't set the `button-label` attribute value to the `nimble-rich-text-mention-users` element in the storybook. As mentioned in [the other comment](https://github.com/ni/nimble/pull/1694#discussion_r1421101129) setting the title and label to this button is a user configuration, so, we cannot see a tooltip or the text in the DOM unless we set it in the element.

Therefore, I added value for the `button-label` on `nimble-rich-text-mention-users` in the storybook and updated the docs for the attribute addition as well.